cmd/opflexagentcni/main.go

```diff
@@ -115,7 +115,7 @@ func waitForAllNetwork(result *current.Result, id string,
 		if err != nil {
 			log.Error("Could not open netns: ", err)
 		} else {
-			waitForNetwork(netns, result, id, index, 10*time.Second)
+			waitForNetwork(netns, result, id, index, timeout)
```

Comment:
I believe the intention earlier was to wait for at least 10 s.

Reply:
Sure both existing callers of this function pass 10 sec as the timeout. The behavior is unchanged by this change

Reply:
I guess an alternative would be to leave hardcoded to 10 sec here and remove the timeout param